src/code/z_map_data.c

```diff
@@ -0,0 +1,389 @@
+    },
+};
+
+static u8 sDgnMinimapCount[12] = {
+    13, 19, 17, 27, 38, 44, 32, 27, 10, 12, 0, 0,
+};
+
+static u16 sDgnMinimapTexIndexOffset[10] = {
+    0, 13, 32, 49, 76, 114, 158, 190, 217, 227,
+};
+
+static u16 sOwMinimapTexSize[24] = {
+    2920, 2560, 1560, 2784, 2976, 2040, 3240, 2336, 2080, 2600, 1792, 1888,
+    3400, 1792, 1888, 2040, 3120, 2304, 2176, 1888, 1560, 3240, 2600, 3400,
+};
+
+static u16 sOwMinimapTexOffset[24] = {
+    0x0000, 0x0B68, 0x1568, 0x1B80, 0x2660, 0x3200, 0x39F8, 0x46A0, 0x4FC0, 0x57E0, 0x6208, 0x6908,
+    0x7068, 0x7DB0, 0x84B0, 0x8C10, 0x9408, 0xA038, 0xA938, 0xB1B8, 0xB918, 0xBF30, 0xCBD8, 0xD600,
+};
+
+static s16 sOwMinimapPosX[24] = {
+    216, 216, 218, 202, 202, 250, 216, 234, 234, 216, 234, 234,
+    216, 234, 234, 250, 216, 234, 234, 234, 218, 80,  80,  216,
+};
+
+static s16 sOwMinimapPosY[24] = {
+    150, 158, 184, 164, 160, 138, 140, 150, 156, 158, 168, 162,
+    138, 168, 162, 138, 146, 150, 154, 162, 218, 81,  65,  216,
+};
+
+static s16 sOwCompassInfo[24][4] = {
+    { 25, 25, 1080, -360 }, { 7, 6, 1000, -650 },   { 6, 6, 890, -800 },    { 7, 7, 720, -730 },
+    { 8, 8, 660, -730 },    { 5, 7, 1220, -660 },   { 13, 13, 1080, -260 }, { 5, 5, 1120, -880 },
+    { 8, 8, 1150, -630 },   { 11, 11, 1060, -680 }, { 12, 12, 1100, -720 }, { 11, 11, 930, -710 },
+    { 9, 9, 850, -830 },    { 15, 15, 1100, -780 }, { 10, 10, 1030, -540 }, { 10, 9, 1240, -700 },
+    { 6, 6, 1030, -620 },   { 5, 5, 1100, -660 },   { 11, 11, 1120, -750 }, { 10, 10, 1070, -580 },
+    { 6, 6, 890, -800 },    { 13, 13, 1080, -260 }, { 11, 11, 1060, -680 }, { 9, 9, 850, -830 },
+};
+
+static s16 sDgnMinimapTexIndexBase[10] = {
+    0, 10, 14, 18, 26, 36, 44, 52, 60, 66,
+};
+
+static s16 sDgnCompassInfo[10][4] = {
+    { 3, 3, 1070, -690 }, { 4, 4, 1070, -690 }, { 3, 3, 1070, -690 },  { 4, 4, 1070, -690 }, { 4, 4, 1070, -690 },
+    { 4, 4, 900, -640 },  { 3, 3, 900, -640 },  { 10, 10, 900, -640 }, { 5, 5, 900, -640 },  { 3, 3, 900, -640 },
+};
+
+static s16 sOwMinimapWidth[24] = {
+    80, 80, 80, 96, 96, 48, 80, 64, 64, 80, 64, 64, 80, 64, 64, 48, 80, 64, 64, 64, 80, 80, 80, 80,
+};
+
+static s16 sOwMinimapHeight[24] = {
+    73, 64, 39, 58, 62, 85, 81, 73, 65, 65, 56, 59, 85, 56, 59, 85, 78, 72, 68, 59, 39, 81, 65, 85,
+};
+
+static s16 sOwEntranceIconPosX[24] = {
+    1, 269, 1, 1, 273, 279, 259, 1, 260, 1, 1, 235, 1, 1, 1, 267, 261, 1, 1, 260, 294, 259, 1, 243,
+};
+
+static s16 sOwEntranceIconPosY[24] = {
+    0, -833, 0, 0, -850, -889, -829, 0, -844, 0, 0, -836, 0, 0, 0, -852, -873, 0, 0, -848, -825, -829, 0, -833,
+};
+
+static u16 sOwEntranceFlag[20] = {
+    0xFFFF, 0x0008, 0x0007, 0xFFFF, 0x0000, 0x0003, 0x0005, 0xFFFF, 0x0002, 0xFFFF,
+    0xFFFF, 0x0006, 0x000B, 0xFFFF, 0xFFFF, 0x0001, 0x0004, 0xFFFF, 0xFFFF, 0x000D,
+};
+
+static f32 sFloorCoordY[10][8] = {
+    { 9999.0, 9999.0, 9999.0, 760.0, 360.0, -40.0, -1000.0, -2000.0 },
```

Comment:
Also pretty minor but we normally format floats as `.0f` unless required otherwise for matching.